libc/src/__support/FPUtil/ManipulationFunctions.h

```diff
@@ -102,7 +103,7 @@ intlogb(U x) {
     return IntLogbConstants<T>::T_MAX;
   }
 
-  DyadicFloat<FPBits<U>::STORAGE_LEN> normal(bits.get_val());
+  DyadicFloat<cpp::max(FPBits<U>::STORAGE_LEN, 32)> normal(bits.get_val());
```

Comment:
`DyadicFloat<Bits>` uses `UInt<Bits>` which requires `Bits` to be either 32 or a multiple of the size of `WordType` which is never `uint16_t`.

Reply:
Maybe update `DyadicFloat` to accept `Bits` to also accept 16?